### PR TITLE
Bench write_b

### PR DESCRIPTION
**Pull request by lu-zero**

### CODE REVIEW

File: benches/bench.rs [hunk 1]
```diff
@@ -90,5 +90,46 @@ fn aom(b: &mut Bencher) {
...
+        mc: mc,
+    };
+
+    let mode = PredictionMode::DC_PRED;
+    let tx_type = TxType::DCT_DCT;
+
+    let sbx = 0;
+    let sby = 0;
+
+/*    for p in 0..1 {
```
Comment (tdaede) [inline benches/bench.rs]: drop commented out loops?

Comment (lu-zero) [reply benches/bench.rs]: Sure

**Comment by mbebenita** [comment]
Do these benchmarks run via [cargo bench](https://doc.rust-lang.org/1.12.1/book/benchmark-tests.html)?

**Comment by lu-zero** [comment]
Yes.

**Pull request merged**